packages/react/src/components/Actions/OneDropdownButton/OneDropdownButton.tsx

```diff
@@ -1,66 +1,85 @@
-import { useI18n } from "@/lib/providers/i18n"
-import { useMemo, useState } from "react"
+import { F0Icon } from "@/components/F0Icon/index.tsx"
 import {
   DropdownInternal,
-  DropdownItemObject,
-  DropdownItemSeparator,
-} from "../../../experimental/Navigation/Dropdown/internal.tsx"
-import { ChevronDown } from "../../../icons/app/index.ts"
-import { cn, focusRing } from "../../../lib/utils.ts"
-import { Action } from "../../../ui/Action/index.tsx"
+  DropdownItem,
+} from "@/experimental/Navigation/Dropdown/internal.tsx"
+import { ChevronDown } from "@/icons/app/index.ts"
+import { useI18n } from "@/lib/providers/i18n"
+import { cn, focusRing } from "@/lib/utils.ts"
+import { Action } from "@/ui/Action"
+import { actionVariants, buttonSizeVariants } from "@/ui/Action/variants.ts"
+import { useMemo, useState } from "react"
 import {
-  actionVariants,
-  buttonSizeVariants,
-} from "../../../ui/Action/variants.ts"
-import { F0Icon } from "../../F0Icon/index.tsx"
-import { OneDropdownButtonSize, OneDropdownButtonVariant } from "./types.ts"
-
-export type OneDropdownButtonItem<T = string> = Pick<
-  DropdownItemObject,
-  "label" | "icon" | "critical" | "description"
-> & {
-  value: T
-}
-
-export type OneDropdownButtonProps<T = string> = {
-  size?: OneDropdownButtonSize
-  items: (OneDropdownButtonItem<T> | DropdownItemSeparator)[]
-  variant?: OneDropdownButtonVariant
-  value?: T
-  disabled?: boolean
-  loading?: boolean
-  onClick: (value: T, item: OneDropdownButtonItem<T>) => void
-}
+  OneDropdownButtonGroup,
+  OneDropdownButtonItem,
+  OneDropdownButtonProps,
+} from "./types.ts"
 
-function isDropdownItem<T>(
-  item: OneDropdownButtonItem<T> | DropdownItemSeparator
+function isDropdownButtonItem<T = string>(
+  item: OneDropdownButtonItem<T> | OneDropdownButtonGroup<T>
 ): item is OneDropdownButtonItem<T> {
   return "value" in item
 }
 
+/**
+ * Normalize the items to an array of DropdownButtonGroup
+ */
+const normalizeItems = (
+  items:
+    | OneDropdownButtonItem[]
+    | OneDropdownButtonGroup[]
+    | OneDropdownButtonGroup
+) => {
+  if (Array.isArray(items)) {
+    // DropdownButtonItem[]
```

Comment:
AI alert 👮

Reply:
😓 I dont know how to interpret that. I wrote every single line of the normalizeFunction and added the comments to see easily which type we are converting to DropdownButtonGroup 😓

Reply:
I'm sorry! This is type of comments AI loves to add. I'm wondering that maybe we can create separate functions to address each type? Not mandatory though